test/fixtures/billing-iam/main.tf

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  billing_roles = ["roles/billing.admin", "roles/billing.viewer"]
+  members       = [var.member1, var.member2]
+
+  member_group_0 = [
+    "serviceAccount:${var.member1}",
+    "serviceAccount:${var.member2}",
+  ]
+
+  member_group_1 = [
+    "serviceAccount:${var.member2}",
+  ]
+
+  member_groups = [local.member_group_0, local.member_group_1]
+
+  # 1 or 2 roles amount can be specified to generate that amount of bindings.
+  # This variability is used to test how the module behaves on configuration updates.
+
+  billing_bindings = zipmap(
+    slice(local.billing_roles, 0, var.roles),
+    slice(local.member_groups, 0, var.roles)
+  )
+}
+
+provider "google" {
+  version = "~> 2.7"
+}
+
+provider "google-beta" {
+  version = "~> 2.7"
+}
+
+#additive
+
+module "iam_binding_billing_accounts_additive" {
```

Comment:
I know it's the existing pattern in this module, but I'd like to move towards the general CFT pattern of having fixtures wrap examples. Let's move all the config here into an example and have the fixture simply invoke the example.

For example: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/test/fixtures/shared_vpc/example.tf#L17

Reply:
After reviewing your comment, I found an already existing billing-iam example here: https://github.com/terraform-google-modules/terraform-google-iam/blob/master/examples/billing_account/main.tf

Also, do you want to use user, group and SA email in the example? We can use the generated the SA email but user and group email will give an error as Terraform will look for a real user/group email. Is it okay if I remove the user and group email from the example?

Reply:
Yeah let's repurpose that one and drop the user/group variables.